src/explore-education-statistics-admin/src/pages/prototypes/PrototypeTableTool.tsx

```diff
@@ -1,11 +1,10 @@
 /* eslint-disable */
 import PrototypePage from '@admin/pages/prototypes/components/PrototypePage';
 import React from 'react';
-import TableTool from '@common/modules/table-tool/components/TableTool';
 import tableBuilderService, {
-  TableDataQuery,
   ThemeMeta,
 } from '@common/modules/full-table/services/tableBuilderService';
+import TableToolWizard from '@common/modules/table-tool/components/TableToolWizard';
 
 const PrototypeTableTool = () => {
   const publicationSlug = undefined; //"pupil-absence-in-schools-in-england";
```

Comment:
Can the example publication slug be removed now?

Reply:
Removed it :)